docs/development/local_setup.md

```diff
@@ -199,33 +174,34 @@ clusterrolebinding.rbac.authorization.k8s.io/front-proxy-client created
 [...]
 ```
 
-This will start all minimally required components of a Kubernetes cluster (`etcd`, `kube-apiserver`, `kube-controller-manager`)
-and an `etcd` Instance for the `gardener-apiserver` as Docker containers.
-
 ℹ️ [Optional] If you want to develop the `SeedAuthorization` feature then you have to run `make ACTIVATE_SEEDAUTHORIZER=true local-garden-up`. However, please note that this forces you to start the `gardener-admission-controller` via `make start-admission-controller`.
 
 To tear down the local Garden cluster and remove the Docker containers, simply run:
 ```bash
 make local-garden-down
 ```
 
-**Using minikube**
+<details>
+  <summary>Alternative: Using a local kubernetes cluster</summary>
 
-Alternatively, spin up a cluster with minikube with this command:
+  Instead of starting a kubernetes API server and etcd as docker containers, you can also opt for running a local kubernetes cluster, provided by e.g. [minikube](https://minikube.sigs.k8s.io/docs/start/), [kind](https://kind.sigs.k8s.io/docs/user/quick-start/) or docker desktop.
```

Comment:
Assuming you're referring to the title and the body of the 'Alternative' sections, I made their title now bold, such that they are more similar to the other sub-headlines. Hopefully that helps a bit?

Reply:
We can keep the bold `summary` sections.

I rather had in mind the following change to visually separate the note (the section that starts with `> Note:`) from the sections before and after:

```diff
diff --git a/docs/development/local_setup.md b/docs/development/local_setup.md
index 5ba4e4452..4022e334c 100644
--- a/docs/development/local_setup.md
+++ b/docs/development/local_setup.md
@@ -185,6 +185,7 @@ make local-garden-down
   <summary><b>Alternative: Using a local kubernetes cluster</b></summary>

   Instead of starting a kubernetes API server and etcd as docker containers, you can also opt for running a local kubernetes cluster, provided by e.g. [minikube](https://minikube.sigs.k8s.io/docs/start/), [kind](https://kind.sigs.k8s.io/docs/user/quick-start/) or docker desktop.
+
   > Note: Gardener requires self-contained kubeconfig files because of a [security issue](https://banzaicloud.com/blog/kubeconfig-security/). You can configure your minikube to create self-contained kubeconfig files via:
   > ```bash
   > minikube config set embed-certs true
@@ -193,6 +194,7 @@ make local-garden-down
   > ```bash
   > minikube start --embed-certs
   > ```
+
 </details>
```

Reply:
Ah, ok, I see what you were referring to. I guess I tried this change before and it didn't really change anything in the rendered markdown doc – pre-formatted blocks like `quote` and code blocks are already separated from the rest of the content like a new paragraph.

I've uploaded a new commit to this PR adding the newlines